internal/clients/nightfall/nightfall.go

```diff
@@ -278,3 +284,34 @@ func (n *Client) ReviewDiff(
 
 	return comments, nil
 }
+
+func filterFileDiffs(fileDiffs []*diffreviewer.FileDiff, fileIncludeList, fileExcludeList []string) []*diffreviewer.FileDiff {
+	filteredFileDiffs := fileDiffs
+	if fileIncludeList != nil && len(fileIncludeList) > 0 {
+		filteredFileDiffs = filterByFilePath(filteredFileDiffs, fileIncludeList, true)
+	}
+	if fileExcludeList != nil && len(fileExcludeList) > 0 {
+		filteredFileDiffs = filterByFilePath(filteredFileDiffs, fileExcludeList, false)
+	}
+	return filteredFileDiffs
+}
+
+func filterByFilePath(fileDiffs []*diffreviewer.FileDiff, regexPatterns []string, include bool) []*diffreviewer.FileDiff {
+	filteredFileDiffs := []*diffreviewer.FileDiff{}
+	for _, fd := range fileDiffs {
+		matched := matchRegex(fd.PathNew, regexPatterns)
+		if (matched && include) || (!matched && !include) {
+			filteredFileDiffs = append(filteredFileDiffs, fd)
+		}
+	}
+	return filteredFileDiffs
+}
+
+func matchRegex(filePath string, regexPatterns []string) bool {
```

Comment:
can we just add a few unit tests for this method? I know it's kind of covered in the TestFilterFileDiffs but would be good to get explicit tests to make sure this piece is doing exactly what we want it to